python_ta/__init__.py

```diff
@@ -358,13 +368,15 @@ def get_file_paths(rel_path: AnyStr) -> Generator[AnyStr, None, None]:
                 yield os.path.join(root, filename)  # Format path, from root.
 
 
-def _verify_pre_check(filepath: AnyStr) -> bool:
+def _verify_pre_check(filepath: AnyStr, allowed_pylint: bool) -> bool:
     """Check student code for certain issues."""
     # Make sure the program doesn't crash for students.
     # Could use some improvement for better logging and error reporting.
     try:
-        # Check for inline "pylint:" comment, which may indicate a student
-        # trying to disable a check.
+        if allowed_pylint:
+            return True
+            # Check for inline "pylint:" comment, which may indicate a student
```

Comment:
This comment should not be indented within the `if`, since it really is describing what the code *below* it is doing.